teaser/data/output/besmod_output.py

```diff
@@ -0,0 +1,432 @@
+"""This module contains function for BESMod model generation"""
+
+import os
+from mako.template import Template
+from mako.lookup import TemplateLookup
+import teaser.logic.utilities as utilities
+import teaser.data.output.modelica_output as modelica_output
+
+
+def export_besmod(
+        buildings,
```

Comment:
Same comments as in project.py, check which are optional, add type hints, write default if they are optional